alibi_detect/utils/tests/datasets.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+from pytest_cases import parametrize
+from alibi_testing.data import get_movie_sentiment_data
+
+# Note: If any of below cases become large, see https://smarie.github.io/python-pytest-cases/#c-caching-cases
+
+
+# Group dataset "cases" by type of data i.e. continuous, binary, categorical, mixed
+class ContinuousData:
```

Comment:
Datasets are grouped together by data type with `pytest_cases`. We could adopt other strategies here, but this might be best left as a future discussion.